### PR TITLE
[minor] Add VERIFYACLs global to bypass verification of ACLs if set to false. Implements #236

### PR DESCRIPTION
**Pull request by jboyer2012**

### CODE REVIEW

File: action.go [hunk 1]
```diff
@@ -343,7 +343,7 @@ func (a Action) String() (str string, err error) {
 // permission is found, the default one `default` is used.
 // The first permission that is found to apply to an operation, but
 // doesn't allow the operation to run, will fail the verification globally
-func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
+func (a Action) VerifyACL(acl ACL, keyring io.Reader, verifyACLs bool) (err error) {
```
Comment (ameihm0912) [inline action.go]: I'm wondering if we should use something other than verifyACLs for the flag that is a bit clearer.

Perhaps onlyVerifyPubkey or something like that?

File: action.go [hunk 2]
```diff
@@ -368,6 +368,14 @@ func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
 		fingerprints = append(fingerprints, fp)
 	}
 
+	if len(fingerprints) == 0 {
+		return errors.New("No valid fingerprints found.")
+	}
+
+	if !verifyACLs {
```
Comment (ameihm0912) [inline action.go]: If we invert the logic above (require true to bypass ACL check) we'd need to also switch this around.

**Comment by ameihm0912** [comment]
Can we rebase against master for this branch too ?

File: action.go (hunk 1 above)
Comment (jboyer2012) [inline action.go]: Do you mean throughout the entire application (in the configuration.go and config.go files) or just the variable passed in to the VerifyACL function?

Comment (ameihm0912) [inline action.go]: Was thinking just here, so if onlyPubkey (or whatever we decide to call it) is true, we only verify the key, but if it's false (default) the ACL code is run too

Comment (jboyer2012) [inline action.go]: I just committed [9ad057e](https://github.com/jboyer2012/mig/commit/9ad057e22962def33050b75aea3f40b7aeaf7519) for these changes. I decided to change the flag everywhere. After thinking about it, I feel it is clearer in its current form. I also reworded the comments to make more explicit what the flag does. Let me know if you like the changes.

Comment (jboyer2012) [inline action.go]: Sorry just realized the last commit I failed to update the actual call to the VerifyAcls function. [d177bd8](https://github.com/jboyer2012/mig/commit/d177bd86dc821d70fe5a1a570d0b478f8bb0dda8) fixed it.

**Comment by ameihm0912** [comment]
Looks good, the travis build is failing as the stand alone install needs to be updated but I can fix that up. Thanks for grabbing this.

**Pull request merged**